portal-impl/src/com/liferay/portal/util/PropsValues.java

```diff
@@ -733,6 +733,10 @@
 	public static final String DATABASE_MYSQL_ENGINE = PropsUtil.get(
 		PropsKeys.DATABASE_MYSQL_ENGINE);
 
+	public static final boolean DDM_DATA_PROVIDER_ACCESS_LOCAL_NETWORK = 
```

Comment:
@yxingwu Can we use DDMFormWebConfiguration to add this setting? Any reason to add it to PropsValue?